docs/source/markdown/podman-buildfarm.1.md

```diff
@@ -0,0 +1,34 @@
+% podman-buildfarm 1
+
+## NAME
+podman\-buildfarm - Farm out builds to machines running podman for different architectures
+
+## SYNOPSIS
+**podman buildfarm** *subcommand*
+
+## DESCRIPTION
+Farm out builds to machines running podman for different architectures.
+
+Manage farms by creating, updating, and removing them.
+
+## COMMANDS
+
+| Command  | Man Page                                                                      | Description                                                |
+| -------- | ----------------------------------------------------------------------------- | ---------------------------------------------------------- |
+| create      | [podman-buildfarm\-create(1)](podman-buildfarm-create.1.md)         | Create a new farm                  |
```

Comment:
Nit: on future PRs can you line up this table.

Reply:
fixed